### DISCUSSION TITLE
How do I get rid of the white flash on refresh (dark mode)?

### DISCUSSION THREAD

**Discussion by 15MariamS** [microsoft/fluentui#30282] [Q&A]
When I refresh my page, there's a white flash as the page renders. How can I get rid of that? (Using tailwind, it's super simple but can't figure it out on fluent)

**Comment by khmakoto** [comment]
Hi @15MariamS, do you have a codepen or codesandbox where I can repro this + repro steps? I can't fully picture what's happening as is 😄
